chart/nats-connector/values.yaml

```diff
@@ -0,0 +1,11 @@
+replicas: 1
+image: openfaas/nats-connector:0.2.2
```

Comment:
helm convention is to use camel case, and also include a comment with each/most values giving some indication of what they are used for in the chart

Reply:
values is kept like that to make it compatible with existing connectors. May be we can create a new issue to update the values across all connectors chart.

cc @alexellis

Reply:
I will add comments for each values.